tests/functests/FuncTests.vcxproj

```diff
@@ -94,7 +94,7 @@
   <ImportGroup Label="Shared">
     <Import Project="..\..\Solutions\Clienttelemetry\Clienttelemetry.vcxitems" Label="Shared" />
     <Import Project="..\..\lib\pal\desktop\desktop.vcxitems" Label="Shared" />
-    <Import Project="..\..\lib\decoder\decoder.vcxitems" Label="Shared" />
+    <!-- Import Project="..\..\lib\decoder\decoder.vcxitems" Label="Shared" / -->

```

Comment:
Why comment? Why not just delete this, as it's a strict subset of ClientTelemetry?